src/styles/PageStyling.scss

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css?family=Bowlby+One+SC');
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+@import url('https://fonts.googleapis.com/css?family=Fira+Code');
+
+.lesson-title {
```

Comment:
looks good, but where do you actually use these class names?

Reply:
I thought I'd make a pull request for with those changes on my creation branch once the styling scss was merged since I did not want to clutter the styling branch. However, I can add them here itself if that's ok?